src/script/interpreter.cpp

```diff
@@ -534,6 +540,18 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 case OP_NOP:
                     break;
 
+                case OP_CAT: {
```

Comment:
```suggestion
                case OP_CAT:
                {
```

Reply:
fixed in [100e207](https://github.com/bitcoin-inquisition/bitcoin/pull/39/commits/100e2076bca252aa712eb11779e85605226f3614)